server/routes/conditions/validationSchemas/selectConditionsSchema.ts

```diff
@@ -3,20 +3,9 @@ import { createSchema } from '../../../middleware/validationMiddleware'
 import { asArray } from '../../../utils/utils'
 import ConditionType from '../../../enums/conditionType'
 import { textValueExceedsLength } from '../../../utils/validation/textValueValidator'
+import conditionsThatRequireNaming from '../conditionsThatRequireNaming'
 
 const selectConditionsSchema = async () => {
-  const CONDITION_TYPES_REQUIRING_NAME = [
```

Comment:
Refactor the validator to use the `conditionsThatRequireNaming` array rather than defining it's own array